### PR TITLE
dev-fix-release-v0.0.3

### PR DESCRIPTION
**Pull request by raphaelmonnard**

### CODE REVIEW

File: fabcal/forms.py [hunk 1]
```diff
@@ -33,25 +33,64 @@ class AbstractSlotForm(forms.Form):
...
             )
 
+        # check if opening slot already exist / for update
+        if 'opening' in self.initial:
+
+            # check if user alread booked a modified slot
+            qs = MachineSlot.objects.filter(opening_slot=OpeningSlot.objects.get(pk=self.initial['id']))
+
+            for obj in qs:
+                if obj.start < self.cleaned_data['start'] and obj.user:
```
Comment (ioknox) [inline fabcal/forms.py:69]: Can be directly filtered in the QuerySet

File: machines/migrations/0005_auto_20221123_0802.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.16 on 2022-11-23 08:02
```
Comment (ioknox) [inline machines/migrations/0005_auto_20221123_0802.py:1]: To be squased ?

**Review by ioknox — COMMENTED** [review]
I think at least the DB squash should be performed prior to merge.

File: machines/migrations/0005_auto_20221123_0802.py (hunk 2 above)
Comment (raphaelmonnard) [reply machines/migrations/0005_auto_20221123_0802.py]: fixed: [638e381](https://github.com/fablab-chaux-de-fonds/Interlab/pull/137/commits/638e3812accae8a710693aae0b6aa1b692a96791)

File: fabcal/templates/fabcal/event_registration_list.html [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
-                <div class="text-center">
-                    <img src="https://eu.ui-avatars.com/api/?rounded=true&name={{registration.first_name}}+{{registration.last_name}}&color=fff&background=e3005c"
-                        class="img w-50" alt="avatar"> </br>
-                    {% else %}
-                    <img src="https://eu.ui-avatars.com/api/?rounded=true&name={{registration.username}}&color=fff&background=e3005c"
-                        class="img w-50" alt="avatar"> </br>
-                    {% endif %}
+            <div class="text-center">
+            {% if event_slot.registration.first_name %}
+                <img src="https://eu.ui-avatars.com/api/?rounded=true&name={{registration.first_name}}+{{registration.last_name}}&color=fff&background=e3005c"
```
[the remark this answers is not in the document]
Comment (raphaelmonnard) [reply fabcal/templates/fabcal/event_registration_list.html:15]: #207

File: fabcal/forms.py (hunk 1 above)
Comment (raphaelmonnard) [reply fabcal/forms.py:69]: #208

**Pull request merged**